src/main/webapp/WEB-INF/web.xml

```diff
@@ -13,8 +13,6 @@
 		<url-pattern>/synapse</url-pattern>
 		<url-pattern>/</url-pattern>
 		<url-pattern>/health</url-pattern>
-		<url-pattern>/about</url-pattern>
```

Comment:
why is this removed?  i think we want to keep it.

Reply:
Yes, you are right. I've updated the PR.